<ui version="4.0" >
 <class>BuscarFacturaDialogClass</class>
 <widget class="QDialog" name="BuscarFacturaDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BuscarFacturaDialog</string>
  </property>
  <property name="windowIcon" >
   <iconset>images/BFactura.PNG</iconset>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>567</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item rowspan="2" row="1" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Fecha:</string>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="1" column="1" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Del</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="1" column="5" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>al</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="1" column="2" colspan="2" >
     <widget class="QDateEdit" name="dateEdit" >
      <property name="displayFormat" >
       <string>dd/MM/yyyy</string>
      </property>
      <property name="calendarPopup" >
       <bool>true</bool>
      </property>
      <property name="date" >
       <date>
        <year>2007</year>
        <month>1</month>
        <day>2</day>
       </date>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="1" column="6" colspan="2" >
     <widget class="QDateEdit" name="dateEdit_2" >
      <property name="displayFormat" >
       <string>dd/MM/yyyy</string>
      </property>
      <property name="calendarPopup" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>NÃºmero</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3" >
     <widget class="QLineEdit" name="lineEdit" />
    </item>
    <item row="0" column="5" colspan="4" >
     <widget class="QLineEdit" name="lineEdit_2" />
    </item>
        <item rowspan="2" row="1" column="4" >
     <widget class="QCheckBox" name="checkBoxDel" >
      <property name="text" >
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="1" column="8" >
     <widget class="QCheckBox" name="checkBoxAl" >
      <property name="text" >
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="5" colspan="2" >
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7" colspan="2" >
     <widget class="QPushButton" name="pushButton_2" >
      <property name="text" >
       <string>Cancelar</string>
      </property>
     </widget>
    </item>

    <item row="0" column="4" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Cliente</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BuscarFacturaDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>503</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel" >
     <x>505</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BuscarFacturaDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>426</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAl</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateEdit_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>542</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel" >
     <x>411</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDel</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel" >
     <x>185</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
